sdk/search/azure-search-documents/src/main/java/com/azure/search/documents/models/GeoPoint.java

```diff
@@ -108,10 +116,14 @@ public int hashCode() {
     @Override
     public String toString() {
         if (isValid()) {
+            String longitude = ("" + coordinates.get(0)).contains(".")
+                ? "" + coordinates.get(0) : "" + coordinates.get(0) + ".0";
+            String latitude = ("" + coordinates.get(1)).contains(".")
+                ? "" + coordinates.get(1) : "" + coordinates.get(1) + ".0";
```

Comment:
`Double.toString(value)` will append `.0`. Don't have to do this explicitly.
```suggestion
            String latitude = Double.toString(coordinates.get(1));
```